package.json

```diff
@@ -78,8 +78,8 @@
         "postcss-nested": "^4.2.3",
         "postcss-simple-vars": "^5.0.2",
         "raw-loader": "^4.0.0",
-        "react": "^17.0.0",
-        "react-dom": "^17.0.0",
+        "react": "^18.0.0",
```

Comment:
as far as i understand, we cannot really use any new `react` features together while still using our outdated version of `mobx` (https://github.com/sulu/sulu/issues/6343). but hopefully it should also not cause any issues 🙂